app/javascript/elm/tests/Data/HeatMapThresholdsTests.elm

```diff
@@ -0,0 +1,27 @@
+module Data.HeatMapThresholdsTests exposing (suite)
+
+import Data.HeatMapThresholds exposing (extremes, updateMaximum, updateMinimum)
+import Expect
+import Fuzz exposing (int, list, string)
+import Json.Encode as Encode
+import Sensor exposing (..)
+import Test exposing (..)
+
+
+suite : Test
+suite =
+    describe "Data.HeatMapThresholds"
+        [ fuzz2 int int "updateMinimum, updateMaximum and extremes" <|
```

Comment:
(note to self?) what happens if h5 is smaller than h1? is this ever the case?

Reply:
I'm answering this as part of the PR conversation ;)